guide/daily-spending-wallet/contacts.md

```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
 
 ### Importing an address
 
-In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the address and offers the user appropriate options. Here, the user adds the address as a new contact.
+This scenario can be initiated by copying a Lightning address to the clipboard, scanning a QR code, or tapping a payment link (see [payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }})). An address is passed into the application and, where it's recognized and appropriate options are shown to the user. In the example below, the user adds the address as a new contact.
 
 {% include image-gallery.html pages = page.imagesImportAddress %}
```

Comment:
This first image should state "You have a lightning address on your clipboard" not "You copied a Lightning address." That sounds like the copy action was taken by the user within their DSW app, not externally which is what this section is covering. 

Also I would have two options in the first frame. One being "Add contact" button (which is already there) which automatically opens into a add new contact frame (avoiding going through the contact list frame). Then a "Add to existing contact" button which goes to the contact list frame where they select an existing contact to add the address to. This removes one extra step to take if you are adding a new contact.